app/controllers/sessions_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
 
     # todo: should we do this everytime?
     calendar = service.get_calendar('primary')
-    user.update!(time_zone: calendar.time_zone.split('/').last.gsub('_', ' '))
+    user.update!(time_zone: calendar.time_zone)
```

Comment:
When I signed up, my time zone was pulling from Google's APIs in the railsy format`America/New_York`.